test/mlir/conversion/onnx_to_torch/matmul1.onnx.mlir

```diff
@@ -0,0 +1,9 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {llvm.data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128", llvm.target_triple = "x86_64-unknown-linux-gnu"}  {
+  func @main_graph(%arg0: tensor<3x3xf32>) -> tensor<3x3xf32> attributes {input_names = ["0"], output_names = ["1"]} {
+    %0 = "onnx.MatMul"(%arg0, %arg0) {onnx_node_name = "MatMul_0"} : (tensor<3x3xf32>, tensor<3x3xf32>) -> tensor<3x3xf32>
```

Comment:
I think it would be nice to have a couple more tests here, as multiplying square matrices is a special case of this op. For instance, two tests with the following input/output dimensions would be nice:
* `(2x3, 3x5) -> 2x5`
* `(2x3x4x5, 2x3x5x4) -> 2x3x4x4`